util/exercise/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "exercise"
-version = "1.0.0"
+version = "1.2.0"
```

Comment:
Let's increment the version sequentially. If this is merged before #694, then this gets to be v1.1.0.